src/components/rangeslider/attributes.js

```diff
@@ -89,5 +89,17 @@ module.exports = {
             'If visible, perpendicular axes will be set to `fixedrange`'
         ].join(' ')
     },
+    perpendicularaxesinitialrange: {
```

Comment:
This is probably a little to verbose for our needs. Attribute name rarely have more than two words smashed together.

The best option that comes to my mind is `xaxis.rangeslider.fixedanchor: false || true`, but `anchor` is used already elsewhere to declare a different thing.

Looking for opinions from @alexcjohnson @cldougl and @chriddyp

Reply:
Yeah, I find it also too verbose but I prefered to use a clear name until we have a better proposal (since anchor is already used).

Reply:
given that the range slider is always an x axis, we could use something like `fixedyrange` - and if we ever make a y axis range slider it can get a `fixedxrange` attribute.

Reply:
It does makes sense, I was not quite sure rangeslider was for x-axis only.